src/python/pants/core/goals/fmt_test.py

```diff
@@ -46,12 +47,12 @@ class FortranFmtRequest(FmtRequest):
 @rule
 async def fortran_fmt(request: FortranFmtRequest) -> FmtResult:
     output = (
-        await Get(Digest, CreateDigest([FORTRAN_FILE]))
+        await Get(Snapshot, Digest, await Get(Digest, CreateDigest([FORTRAN_FILE])))
```

Comment:
```suggestion
        await Get(Snapshot, CreateDigest([FORTRAN_FILE]))
```
These rules can be chained.